contracts/curation/Curation.sol

```diff
@@ -463,4 +530,28 @@ contract Curation is CurationV1Storage, GraphUpgradeable, ICuration {
             rewardsManager.onSubgraphSignalUpdate(_subgraphDeploymentID);
         }
     }
+
+    /**
+     * @dev Calculate reserve ratio based on initialization phase
+     * @param _createdAt When NameCurationPool was created
+     * @return Reserve ratio
+     */
+    function _getEffectiveReserveRatio(uint256 _createdAt) private view returns (uint32) {
+        uint32 effectiveReserveRatio = defaultReserveRatio;
```

Comment:
One issue I see with reading directly from `defaultReserveRatio` is that if Governance changes the reserve ratio at any point all the calculations from previously initialized subgraphs will change. 

The current behaviour is to store the reserve ratio to use in `Subgraph.reserveRatio` whenever it is initialized. I believe we should still do that during the first mint and use it from here.

What do you think?